stdlib/public/Python/Python.swift

```diff
@@ -228,14 +228,20 @@ extension PythonError : Equatable {
 extension PythonError : CustomStringConvertible {
   public var description: String {
     switch self {
-    case .exception(let p): return "exception: \(p)"
-    case .invalidCall(let p): return "invalidCall: \(p)"
-    case .invalidMember(let m): return "invalidMember: \(m)"
-    case .invalidModule(let m): return "invalidModule: \(m)"
+    case .exception(let p): return "Python Exception: \(p)"
+    case .invalidCall(let p): return "Python Invalid Call: \(p)"
+    case .invalidMember(let m): return "Python Invalid Member: \(m)"
+    case .invalidModule(let m): return "Python Invalid Module: \(m)"
     }
   }
 }
 
+extension PythonError: LocalizedError {
```

Comment:
Add space before colon. The TensorFlow module and the Python module follow the standard library style: there’s a space before colons in places where the colon stands for conformances/inheritance.

Reply:
any chance we will have a codified style for this project? i know the standard library uses a style but its more add-hock

Reply:
It'd be good to converge with the Swift community on a style guide for stdlib, but we don't have a timeline. Currently it's more like an implicit convention.